src/aura/HH_Container/HH_ContainerHelper.js

```diff
@@ -106,7 +106,9 @@
 
             // tell our visualforce page we are done loading
             var event = $A.get("e.c:HH_ContainerLoadedEvent");
-            event.fire();
```

Comment:
Doug's pattern is basically, pass an object with callback functions to a lightning component, so that component can call your visualforce javascript code.  The fact that he put it on the app component, and I am putting it on my main Lightning component is a design decision I made to be more direct and consistent with my Lightning container component being the brains behind the whole page.  In fact, it was the one firing the three events that the visualforce page needed.  I decided to keep the same event prototype to be consistent with lightning events, so my visualforce handlers could also use the same event object to receive parameters.  So I want to continue with the design I have.